modules/dnn/include/opencv2/dnn/dnn.hpp

```diff
@@ -339,6 +339,13 @@ CV__DNN_EXPERIMENTAL_NS_BEGIN
     public:
 
         CV_WRAP Net();  //!< Default constructor.
+        /** @brief Create a network from Intel's Model Optimizer intermediate representation.
+         *  @param[in] xml XML configuration file with network's topology.
+         *  @param[in] bin Binary file with trained weights.
+         *  Networks imported from Intel's Model Optimizer are lauched in Intel's Inference Engine
+         *  backend.
+         */
+        CV_WRAP Net(const String& xml, const String& bin);
```

Comment:
There are a lot of frameworks uses scheme+weights pattern, so this "two strings" constructor would be very ambiguous, especially in Python or other bindings.
I believe it is better to follow existed pattern and add separate static method like `readDLDT(const String& xml, const String& bin)` instead.

Reply:
@alalek, Thanks! Replaced to `CV_WRAP static Net readFromModelOptimizer(const String& xml, const String& bin);`